packages/select/src/__tests__/examples.tsx

```diff
@@ -1,10 +1,11 @@
 import type { DummyItem } from './constants';
```

Comment:
this is minor, but i usually see stuff like this being referred to as 'fixtures', since it seems this is used in a test. As opposed to like an example test, or a story for Storybook.